src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,41 @@
+package duke.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import duke.commands.Command;
+import duke.commands.CommandType;
+import duke.exceptions.DukeInvalidCommandException;
+import duke.tasks.TaskList;
```

Comment:
Good spacing and ordering of imported packages. I like how you also separated static imports from normal ones. Good job for doing the optional Checkstyle for IP (I assume you did :>).